### ISSUE TITLE
Data provider metadata in multiple languages

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1549]
**Data provider metadata in multiple languages**

Now that you can present the interface in multiple languages, we now require that data providers share metadata in multiple languages. For example, the Canadian Museum of Nature is also officially known as "Musée canadien de la nature". But, we've got no mechanism to tell you that. The presentation on `https://www.gbif.org/fr/publisher/a41250f0-7c3e-11d8-a19c-b8a03c50a862` is not pleasing to our stakeholders.

-----

User provided contact info: @dshorthouse
System: Chrome 69.0.3497 / Mac OS X 10.13.6
User: [See in registry](https://www.gbif.org/api/feedback/user/724448b07627f95b9af44d26fbab1541:c05d46ecb2df8485c0aa12dae635ad946733a83b07f1438121a41741a74874dcc4e34a19928d284dda8bff26aa9a52742d161a343e6fe778022e0c1af576d63132a4ca3229c3102a1262c64ff4b60dd5)
Referer: https://www.gbif.org/publisher/a41250f0-7c3e-11d8-a19c-b8a03c50a862
Window size: width 1434 - height 748
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-09-20T14:02:58.757Z',mode:absolute,to:'2018-09-20T14:08:58.757Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-09-20T14:02:58.757Z',mode:absolute,to:'2018-09-20T14:08:58.757Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by MattBlissett** [comment]
I just came across this: https://github.com/NCEAS/eml#internationalization-in-eml-211

EML 2.1.1 supports multilingual elements. We don't support that yet (not in the IPT, not in data ingestion).

**Comment by cgendreau** [comment]
This is exactly what I was looking for.
We would also like to provide Title and Abstract in 2 different languages in EML so that the dataset page could display them in an alternative language.

**Comment by peterdesmet** [comment]
In favour of offering publishers the option to provide at least the dataset title and description in a language other than English or in addition to (a maybe required) English. Note that this partly affects "Metadata language". Do any datasets use something else than English?

Also, note that the originally issue description was about the description of a publisher: https://www.gbif.org/fr/publisher/a41250f0-7c3e-11d8-a19c-b8a03c50a862. Multiple language support for that should done at the registry.gbif.org level.

**Comment by cgendreau** [comment]
Indeed, the original issue was about publisher and even if related, it's a different issue.
